app/assets/stylesheets/application.scss

```diff
@@ -168,3 +168,7 @@ h6 {
    opacity : 0.4;
    z-index: -1;
 }
+
+a {
+  color: #497714;
```

Comment:
Color literals like `#497714` should only be used in variable declarations; they should be referred to via variable everywhere else.